### PR TITLE
[RFR]Fixing volume snapshot tests for openstack in 5.11

### PR DESCRIPTION
**Pull request by mmojzis**
Volume fixture has to use availability zone in 5.11 for testing snapshots.

### CODE REVIEW

**Comment by dajoRH** [comment]
I detected some fixture changes in commit 6070b10c2a8da425c8e4034ce7e3828011125367

The *local* fixture **`volume`** is used in the following files:
- **cfme/tests/storage/test_volume_snapshot.py**
    - *snapshot*
    - *test_storage_snapshot_create_cancelled_validation*
    - *test_storage_snapshot_create_reset_validation*
    - *test_storage_volume_snapshot_crud*

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

File: cfme/tests/storage/test_volume_snapshot.py [hunk 1]
```diff
@@ -32,11 +32,18 @@ def volume(appliance, provider):
...
-        volume = volume_collection.create(name=name,
-                                          tenant=provider.data['provisioning']['cloud_tenant'],
-                                          volume_size=STORAGE_SIZE,
-                                          provider=provider,
-                                          cancel=False)
+        if appliance.version >= "5.11":  # has mandatory availability zone
+            volume = volume_collection.create(name=fauxfactory.gen_alpha(),
+                                              tenant=provider.data['provisioning']['cloud_tenant'],
+                                              volume_size=STORAGE_SIZE,
+                                              az=provider.data['provisioning']['availability_zone'],
```
Comment (valaparthvi) [inline cfme/tests/storage/test_volume_snapshot.py]:
```suggestion
volume = volume_collection.create(name=fauxfactory.gen_alpha(),
                                              tenant=provider.data['provisioning']['cloud_tenant'],
                                              volume_size=STORAGE_SIZE,
                                              az=provider.data['provisioning']['availability_zone'] if appliance.version > "5.11" else None,
                                              provider=provider)
```

Comment (mmojzis) [reply cfme/tests/storage/test_volume_snapshot.py]: I reworked it and it uses kwargs instead now. @valaparthvi

**Comment by dajoRH** [comment]
I detected some fixture changes in commit e76b2608365c74737ea659ea87e675675b40b76d

The *local* fixture **`volume`** is used in the following files:
- **cfme/tests/storage/test_volume_snapshot.py**
    - *snapshot*
    - *test_storage_snapshot_create_cancelled_validation*
    - *test_storage_snapshot_create_reset_validation*
    - *test_storage_volume_snapshot_crud*

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

**Pull request merged**